OpenRA.Mods.Cnc/FileFormats/VqaVideo.cs

```diff
@@ -160,12 +146,24 @@ public VqaReader(Stream stream)
 
 			CollectAudioData();
 
+			if (useFramePadding)
+			{
+				var frameSize = Exts.NextPowerOf2(Math.Max(Width, Height));
+				CurrentFrameData = new byte[frameSize * frameSize * 4];
+				totalFrameWidth = (ushort)frameSize;
+			}
+			else
+			{
+				CurrentFrameData = new byte[Width * Height * 4];
```

Comment:
This will probably crash when Width/Height are not power of 2.

Reply:
As with the above comment, currently all callers are related to `VideoPlayerWidget`, which expects padding to be present, so they all pass `true` and go through the `if`. `VideoPlayerWidget` would indeed crash on this `else`, but creation of a PNG for example doesn't expect padding so that should be fine.